src/proto/mgmt/system.proto

```diff
@@ -83,3 +83,20 @@ message SystemEraseReq {
 message SystemEraseResp {
 	repeated shared.RankResult results = 1;
 }
+
+// SystemCleanupReq supplies the machinename.
+message SystemCleanupReq {
+	string sys = 1; // DAOS system identifier
+	string machine = 2; // Name of the machine whos resources need cleanup
+	bool verbose = 3; // Whether response should contain a list of pools and handles evicted
```

Comment:
Hmm. This seems a little strange. Shouldn't we just always return all of the info and let the caller decide what to display?

Reply:
I figured it would be less work on the data path but I can change it to let the client side decide.